Add docs for custom interact action

Rio02coder opened the pull request:
# Pull Request for Documentation Update

## Description
Provide a summary of the changes made to the documentation.

## Related Issues
Link to any related issues or discussions (e.g., `Fixes #123`).

## Changes Made
- [x] Updated existing documentation
- [ ] Added new documentation
- [ ] Fixed typos or formatting issues

## Checklist
- [ ] I have reviewed the changes for accuracy.
- [ ] I have ensured proper formatting and structure.
- [ ] I have tested the changes locally (if applicable).

## Additional Notes
Add any additional notes or context about the changes here.

Commit:
Add docs for custom interact action

ChrisIsKing commented on jvdocs.trueselph.com/docs/actions/CustomInteractAction.md:
````diff
@@ -0,0 +1,98 @@
+actions/anonymous/<action_name>
+```
+
+## Bug Fix
````
Remove this, it has been patched in the jvcli

ChrisIsKing reviewed (commented):
This is a decent start. To be more comprehensive I would add the following:

- A breakdown of each of the files generated from the jpr and their respective purpose.
- Visuals of adding the action in the jvcli client, configuring it's properties, showing the action on the graph etc.

Feel free to be as comprehensive as possible, we can always trim later. You can also may restructure it to be more of a tutorial where you build a simple interact action out.

Commit:
Custom Interact Action Docs

- Add graph structure of action nodes to custom interact action docs
- Add a concrete custom interact action node example
